tutendero/src/main/kotlin/com/tutendero/api/security/jwt/InvalidJwtAuthenticationException.kt

```diff
@@ -0,0 +1,12 @@
+package com.tutendero.api.security.jwt
+
+import org.springframework.security.core.AuthenticationException
+
+class InvalidJwtAuthenticationException(e: String?) : AuthenticationException(e) {
```

Comment:
should this have differents options of constructor?
remember It ca be done using word constructor in kotlin